docs/src/pages/ui/components/authenticator/index.page.mdx

```diff
@@ -105,6 +105,37 @@ can sign in with Google, Amazon or Facebook.
 
 <Feature name="sign-in-federated" />
 
+### Internationalization (I18n)
+
+The `Authenticator` ships with [translations](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts) for:
+
+- `en` – English (default)
+- `zh` – Chinese
+- `fr` – French
+- `de` – German
+- `it` – Italian
+- `ja` – Japenese
+- `es` – Spanish
+
+These [translations](https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/i18n/translations.ts) can be customized using the [Amplify JS' `I18n`](https://docs.amplify.aws/lib/utilities/i18n/q/platform/js/) module:
+
+```js
+I18n.setLanguage('fr');
+
+I18n.putVocabularies({
+  fr: {
+    'Sign In': 'Se connecter',
+    'Sign Up': "S'inscrire",
+  },
+  es: {
+    'Sign In': 'Registrarse',
+    'Sign Up': 'Regístrate',
+  },
+});
```

Comment:
I agree, that would be consistent with these docs (and better for customers):
> https://docs.amplify.aws/ui/customization/translations/q/framework/angular/

Admittedly, this snippet came directly from https://docs.amplify.aws/lib/utilities/i18n/q/platform/js/#putvocabularies 🙈 

We'll likely want to have consistency between exports (e.g. `Translations` instead of `DefaultTexts`), but with this being authenticator-only, perhaps we should export `AuthenticatorTranslations` or similar 🤔

Reply:
Yeah, I wasn't sure of `DefaultTexts` either. The only reason I didn't use `Translations` was because the variable name `translations` was already taken: https://github.com/aws-amplify/amplify-ui/blob/7ccea320be5b62a0050cef9464a64536a1bf6b25/packages/ui/src/i18n/translations.ts#L101-L103.

Now that we're providing translation by default, maybe we rename that `translations` to something else so `Translations` isn't confuisng.

In terms of authenticator-only, we can name it `AuthenticatorTranslations`, then combine with other error string Amplify provides like [`AuthErrorStrings`](https://github.com/aws-amplify/amplify-js/blob/main/packages/auth/src/common/AuthErrorStrings.ts#L1). 


```ts
import { AuthErrorStrings } from 'aws-amplify';
export const Translations = {...AuthenticatorTranslations, ...AuthErrorStrings}
```